sprokit/processes/core/associate_detections_to_tracks_process.cxx

```diff
@@ -126,9 +126,16 @@ ::_step()
   {
     scoped_step_instrumentation();
 
-    // Run associator
-    d->m_track_associator->associate( frame_id, image,
-           tracks, detections, ass_matrix, output, unused );
+    try
+    {
+      // Run associator
+      d->m_track_associator->associate( frame_id, image,
+            tracks, detections, ass_matrix, output, unused );
+    }
+    catch ( vital::invalid_value const& e )
+    {
+      LOG_ERROR( logger(),  "An error occurred: " << e.what());
```

Comment:
Would edit the message to have a more specific description of the error. For example, `"Failed to associate tracks: " << ...`.
Also, you have an extraneous space following `logger(),  ` :)

Reply:
Thanks a lot for the feedback! I've implemented the suggested changes :)